tests/testthat/test-clone.R

```diff
@@ -1077,6 +1077,28 @@ test_that("Cloning inherited methods for non-portable classes", {
   expect_identical(a$x, 3)
 })
 
+test_that("In deep_clone(), don't try to clone non-R6 objects", {
+
+  `$.asdfasdf` <- function(x, value) {
+    stop("error")
+  }
+
+  bar <- R6Class("bar",
+                 public = list(
+                   x = NULL,
+                   initialize = function() {
+                     x <- new.env(parent = emptyenv())
+                     class(x) <- "asdfasdf"
+                     self$x <- x
+                     x
+                   }
+                 )
+  )
+
+  obj <- bar$new()
+  obj2 <- obj$clone(deep = TRUE)
+  expect_equal(obj$x, obj2$x)
```

Comment:
I think this should be `expect_identical`?

```
x <- new.env()
y <- new.env()

expect_equal(x, y) # Passes

expect_identical(x, y)
#> Error: `x` not identical to `y`.
#> Objects equal but not identical
```

Reply:
Done